tests/end-to-end/backup-restore-test/backupe2e/statefulset.go

```diff
@@ -238,7 +238,7 @@ func (t *statefulSetTest) waitForPodDeployment(namespace string, replicas int32,
 			}
 
 			if statefulSet.Status.ReadyReplicas != int32(len(pods.Items)) {
-				return fmt.Errorf("Number of pods %v different from ready replicas %v", len(pods.Items), statefulSet.Status.ReadyReplicas)
+				break
```

Comment:
Can we merge this https://github.com/kyma-project/kyma/blob/dc6a72059b357e2a2f40efa3f06a8e933e2b44fd/tests/end-to-end/backup-restore-test/backupe2e/statefulset.go#L232 with this line?